Automattic/wp-calypso
# Jetpack Search: Fix setting check for Jetpack site

Related to https://github.com/Automattic/jetpack/issues/29374.

## Proposed Changes

Fixes detection of whether Jetpack Search is enabled for a Jetpack site.

## Testing Instructions



On WordPress.com:

* Open the live branch for this PR.
* Go to `/jetpack-search/:siteSlug` for a Jetpack site **without a Jetpack Search subscription**. 
Ensure that the page correctly detects the subscription. 
* Go to `/jetpack-search/:siteSlug` for a Jetpack site **with a Jetpack Search subscription**. 
Ensure that the page correctly detects the subscription. 
* Go to `/jetpack-search/:siteSlug` for a simple site **without a Jetpack Search subscription**. 
Ensure that the page correctly detects the subscription. 
* Go to `/jetpack-search/:siteSlug` for a simple site **with a Jetpack Search subscription**. 
Ensure that the page correctly detects the subscription.

On Jetpack Cloud:

* Spin up a local instance using `yarn start-jetpack-cloud`
* Open `http://jetpack.cloud.localhost:3000/jetpack-search/:siteSlug` for a Jetpack site **without a Jetpack Search subscription**. Ensure that the page correctly detects the subscription. 
* Open `http://jetpack.cloud.localhost:3000/jetpack-search/:siteSlug` for a Jetpack site **with a Jetpack Search subscription**. Ensure that the page correctly detects the subscription. 

## Pre-merge Checklist





---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203986291851623
